expr/func_aggregate.go

```diff
@@ -37,19 +52,38 @@ func (s *FuncAggregate) Exec(dataMap DataMap) ([]models.Series, error) {
 		return nil, err
 	}
 
+	agg := seriesAggregator{function: getCrossSeriesAggFunc(s.name), name: s.name}
+	series = Normalize(dataMap, series)
```

Comment:
This can now be called with 0 length input `series`, which causes a crash. I think that `Normalize` should be enhanced to return the input for `len(in) < 2` (since no normalization needs to be done in these cases).